### PR TITLE
New package: GoHarbor.Harbor version 0.0.1

### PR DESCRIPTION
**Pull request by ambahman**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/144972)

### CODE REVIEW

**Comment by amands98** [comment]
@microsoft-github-policy-service agree company="Harbor"

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-17T17:41:12.3281452Z ##[error] One or more errors occurred. ((Line: 12, Col: 29, Idx: 422) - (Line: 12, Col: 48, Idx: 441): While parsing a block mapping, did not find expected key.) (Line: 12, Col: 29, Idx: 422) - (Line: 12, Col: 48, Idx: 441): While parsing a block mapping, did not find expected key.

(Automated response - build 870.)

File: manifests/g/GoHarbor/Harbor/0.0.1/GoHarbor.Harbor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: GoHarbor.Harbor
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Harbor
+PackageName: Harbor CLI
+PackageURL: https://goharbor.io/
+License: Apache License 2.0
+LicenseUrl: https://github.com/goharbor/harbor-cli/blob/main/LICENSE
+ShortDescription: [Sandbox] Official Harbor CLI
```
Comment (stephengillie) [inline manifests/g/GoHarbor/Harbor/0.0.1/GoHarbor.Harbor.locale.en-US.yaml:12]:
```suggestion
ShortDescription: \[Sandbox\] Official Harbor CLI
```

It seems that square brackets are reserved characters in YAML.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-03-24T13:21:59.6189267Z ##[error] Manifest Error: All field names should be PascalCased. [PackageURL]

(Automated response - build 873.)

File: manifests/g/GoHarbor/Harbor/0.0.1/GoHarbor.Harbor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: GoHarbor.Harbor
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Harbor
+PackageName: Harbor CLI
+PackageURL: https://goharbor.io/
```
Comment (stephengillie) [inline manifests/g/GoHarbor/Harbor/0.0.1/GoHarbor.Harbor.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://goharbor.io/
```

**Comment by stephengillie** [comment]
Verify: 
version 0.0.1

**Review by stephengillie — APPROVED** [review]

**Pull request merged**